spec/models/application_proceeding_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+require 'spec_helper'
```

Comment:
✏️ The `rails_helper` already calls the `spec_helper` so this require seems unnecessary.